backend/src/app/services/parcelDescriptions/parcelDescriptions.service.ts

```diff
@@ -236,7 +242,106 @@ export class ParcelDescriptionsService {
     parcelDescriptions: ParcelDescriptionInputDTO[],
     userInfo: any,
   ) {
-    // TODO: Implementation to come in another pull request
+    this.sitesLogger.log(
+      'parcelDescriptionService.addParcelDescriptionsForSite() start',
+    );
+    this.sitesLogger.debug(
+      'parcelDescriptionService.addParcelDescriptionsForSite() start',
+    );
+
+    const now = new Date();
+    // Parse the parcel description into a subdivision database entity.
+    const subdivisions: Subdivisions[] = parcelDescriptions.map(
+      (parcelDescription) => {
+        const pid =
+          parcelDescription.descriptionType === 'Parcel ID'
+            ? parcelDescription.idPinNumber
+            : null;
+        const pin =
+          parcelDescription.descriptionType === 'Crown Land PIN'
+            ? parcelDescription.idPinNumber
+            : null;
+        const crownLandsFileNo =
+          parcelDescription.descriptionType === 'Crown Land File Number'
```

Comment:
if possible can we use Enum instead of hard coded string 'Crown Land File Number' ?